### PR TITLE
Release 1.15.1 Hotfix

### PR DESCRIPTION
**Pull request by andescu**
Updated CHANGELOG and CMakeLists files for the HOTFIX release v1.15.1

Relates-To: OLPEDGE-2759, OLPSUP-19082

Signed-off-by: Andrei Popescu <andrei.popescu@here.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## v1.15.1 (16/06/2022)
+
+**olp-cpp-sdk-dataservice-read**
+* **Hotfix** Reverted pull requests #1332, and #1332 as they introduced a regression which was not caught by our test suite.
```
Comment (andrey-kashcheev) [inline CHANGELOG.md:4]:
```suggestion
* **Hotfix** Reverted pull requests #1332, and #1338 as they introduced a regression which was not caught by our test suite.
```

Comment (andescu) [reply CHANGELOG.md]: Thanks!

**Review by andrey-kashcheev — DISMISSED** [review]

**Review by mykolamalik — DISMISSED** [review]

**Review by dudnyk — DISMISSED** [review]

**Review by mykolamalik — APPROVED** [review]

**Comment by ystefinko** [comment]
Gitlab-bot - Internal OCM verification passed !

**Comment by ystefinko** [comment]
Gitlab-bot - Internal OCM verification passed !

**Pull request merged**